accord-core/src/main/java/accord/topology/TopologyManager.java

```diff
@@ -469,6 +554,35 @@ public TopologyManager(TopologySorter.Supplier sorter, Agent agent, Id self, Sch
         this.localConfig = localConfig;
     }
 
+    public EpochsSnapshot epochsSnapshot()
+    {
+        ImmutableList.Builder<EpochsSnapshot.Epoch> builder = ImmutableList.builder();
```

Comment:
Suggestion: when we know the size of the contents and are already wrapping the contents in some object, I prefer to use a raw array. If not, we should provide the size to the builder.

Reply:
TIL: `ImmutableList.builderWithExpectedSize`